tests/opensearch/basic.bats

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bats
+
+load "$HELPERS_ROOT/test-helpers.bash"
+
+ensure_variables_set BATS_SUPPORT_ROOT BATS_ASSERT_ROOT BATS_DETIK_ROOT BATS_FILE_ROOT TEST_NAMESPACE FLUENTBIT_IMAGE_TAG OPENSEARCH_IMAGE_TAG
+
+load "$BATS_DETIK_ROOT/utils.bash"
+load "$BATS_DETIK_ROOT/linter.bash"
+load "$BATS_DETIK_ROOT/detik.bash"
+load "$BATS_SUPPORT_ROOT/load.bash"
+load "$BATS_ASSERT_ROOT/load.bash"
+load "$BATS_FILE_ROOT/load.bash"
+
+setup() {
+    echo "recreating namespace $TEST_NAMESPACE"
+    if [[ "${SKIP_DELETE_NAMESPACE:-no}" != "yes" ]]; then
+        run kubectl delete namespace "$TEST_NAMESPACE"
+    fi
+    run kubectl create namespace "$TEST_NAMESPACE"
+}
+
+teardown() {
+    if [[ "${SKIP_DELETE_NAMESPACE:-no}" != "yes" ]]; then
```

Comment:
Think it is better to have a single `SKIP_TEARDOWN` that is always used in all tests, regardless of what the teardown is as makes it easier then.

Reply:
ack